runtime/tests/checker/access_test.go

```diff
@@ -1999,7 +1999,9 @@ func TestCheckAccessSameContractInnerStructField(t *testing.T) {
 			if expectSuccess {
 				require.NoError(t, err)
 			} else {
-				require.Error(t, err)
+				errs := ExpectCheckerErrors(t, err, 1)
+
+				assert.IsType(t, &sema.InvalidAccessError{}, errs[0])
```

Comment:
Why do these not use the new `RequireError` function?

Reply:
Refactored in [0e5b5f7](https://github.com/onflow/cadence/pull/2034/commits/0e5b5f7cdcabf6e315cdea76120a3f7b7326ea93)